index.html

```diff
@@ -2238,13 +2238,15 @@ <h5>Resource List</h5>
 
 						<p>The order of items is <em>not significant</em>.</p>
 
-						<p>The URLs MUST NOT include fragment identifiers. It is RECOMMENDED to use
-								<code>LinkedResource</code> objects with their <code>encodingFormat</code> values
-							set.</p>
+						<p>To avoid conflicting information about a resource, a particular resource's <a>URL</a> SHOULD
```

Comment:
Actually, note to myself: at the moment, the context file does turn resources and lists into (RDF) lists, ie, make the order significant. This should be changed (and I will)

Reply:
My mistake. The context file was o.k., there is nothing to change. What was (is) indeed buggy is my memory, but there is no way to change that:-(